rct229/data/schema_enums.py

```diff
@@ -45,10 +63,10 @@ def print_schema_enums():
     """
     for key in schema_enums:
         print(f"{key}:")
-        for e in schema_enums[key]:
-            print(f"    {e.name}: {e.value}")
+        for e in schema_enums[key].get_list():
+            print(f"    {e}")
         print()
 
 
 # Uncomment this for checking the enumerations after a schema change
-# print_schema_enums()
+print_schema_enums()
```

Comment:
Great solution. Thanks for working on it. 
One comment, should this line be commented out?

Reply:
Good catch!  Done.